### ISSUE TITLE
Exercise Type

### ISSUE THREAD

**Issue by mcos** [robinpowered/swolebot#2]
Could you make this configurable for different exercises? Or configure it to make a randomized call to an array of configured exercises? e.g. `['pushups', 'situps', 'burpees', 'take a day off and eat cheetos']`

**Comment by jutaz** [comment]
Yeah, I had idea to do this, but wanted to take some rest during weekend ;) This will be next weekend`s goal

**Comment by jstruzik** [comment]
Support for `keystrokes` as an exercise pls.

**Comment by ghanbak** [comment]
`lunges` pls
